src/test/resources/application.conf

```diff
@@ -21,6 +21,7 @@ consignmentapi = {
     driver = org.h2.Driver
     connectionPool = disabled
     keepAliveConnection = true
+    port = 5432
```

Comment:
@Lholbourn  added this as an alternative solution last week: https://github.com/nationalarchives/tdr-consignment-api#testing-within-intellij

Which do you think we should go with?

Reply:
Lets go with setting the -Dconfig.file variable. It doesn't involve introducing variables that don't do anything like this one does. I've taken these lines out.